Gemfile.lock

```diff
@@ -95,4 +95,5 @@ DEPENDENCIES
   jekyll-last-modified-at
 
 BUNDLED WITH
-   2.2.19
+MSDefenderFW
+   2.2.16
```

Comment:
Are you able to unstage this file from the pull request (or rebase from develop)? It's out of date and will change again in the future anyway. Otherwise, the rest look good to me.

Reply:
@zedzedzed updated as requested